src/main/java/com/jmb/springfactory/model/dto/StatisticDto.java

```diff
@@ -0,0 +1,22 @@
+package com.jmb.springfactory.model.dto;
+
+import java.util.Map;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class StatisticDto {
+
+    private Map<String, Long> priority;
+    private Map<String, Long> status;
+    private Map<String, Long> estimatedHours;
+    private Map<String, Long> workedHours;
+    private Map<String, Long> taskByGroup;
+    private Map<String, Long> workedHoursByGroup;
+    private Map<String, Long> commentByGroup;
+    private Map<String, Long> commentByTask;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "commentByTask" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1068)